test/numerical/TestRNN.cpp

```diff
@@ -107,9 +107,11 @@ bool isOMRNNTheSameAsNaiveImplFor(const int direction, const int S, const int B,
 
   // Emit the entry point operation which specifies the number of user
   // inputs and outputs.
+  std::string signature("");
```

Comment:
I added a test: TestMatMul2D, do you mind to add the same thing to it. That is probably why the compilation failed...

Reply:
done